healme.lic

```diff
@@ -69,32 +73,44 @@ class HealMe
 
   def heal_wound(spells, part, internal)
     if @using_waggle
-      spells['Heal Wounds']['cast'] = "cast #{part}"
-      spells['Heal Wounds']['cast'] = "cast #{part} internal" if internal
-      check_discern(spells['Heal Wounds'], @settings) if spells['Heal Wounds']['use_auto_mana']
+      spells['Heal Wounds']['cast'] = "cast #{part.first['part']}"
+      spells['Heal Wounds']['cast'] = "cast #{part.first['part']} internal" if internal
```

Comment:
💡  What do you think about `spells['Heal Wounds']['cast'] += " internal" if internal`?